src/components/Button/Button.tsx

```diff
@@ -1,18 +1,16 @@
 import * as React from 'react';
 import * as style from './Button.scss';
 
-interface IOwnProps {
-  handleClick: () => void;
+export interface IButtonProps {
+  onClick: () => void;
   text: string;
 }
 
-type IProps = IOwnProps;
-
-export const Button = (props: IProps) => {
-  const { handleClick, text } = props;
+export const Button = (props: IButtonProps) => {
+  const { onClick, text } = props;
   return (
-    <div className={style.button} onClick={handleClick}>
+    <button className={style.button} onClick={onClick}>
```

Comment:
Not sure about using button-tags. Sanitize.css *should* handle most of cross-browser issues, but in general I prefer making it from scratch to avoid any potential issues. It's really not a lot more work

Reply:
For semantics and tab support